_episodes/02-filedir.md

```diff
@@ -420,15 +420,19 @@ without it,
 the shell thinks we're trying to run a command called `ls-F`,
 which doesn't exist.
 
-> ## Parameters vs. Arguments
+> ## Flag or Option vs. Parameters or Arguments
 >
-> According to [Wikipedia](https://en.wikipedia.org/wiki/Parameter_(computer_programming)#Parameters_and_arguments),
-> the terms **argument** and **parameter**
-> mean slightly different things.
-> In practice,
-> however,
-> most people use them interchangeably or inconsistently,
-> so we will too.
+> Although there may be technical differences between the terms, Flag and Option,
```

Comment:
How about rephrasing as:

Although there may be technical differences between the terms "flags" (or "options") and "parameters" (or "arguments"), most people use them interchangeably and sometimes inconsistently.

Reply:
Sorry, @shwina  but I think this muddies the waters rather than clarifying. Most people don't use arguments and flags interchangeably -- it's parameter and arguments that get confused.